qiskit_aer/backends/aer_compiler.py

```diff
@@ -823,6 +823,14 @@ def _assemble_op(
     }:
         aer_circ.gate(name, qubits, params, [], conditional_reg, aer_cond_expr,
                       label if label else name)
+    elif name == "cx_o0":
```

Comment:
Shouldn't we check this for other gates as well? The same would be applicable to `cy`, `cz` and other gates (with ctrl_state arg) right?